.travis.yml

```diff
@@ -2,6 +2,7 @@ sudo: false
 dist: trusty
```

Comment:
You should remove this one if switching to 18.04

Reply:
good catch, I was actually looking for this but oversaw it ...

Do you see any problems with switching to bionic?

Reply:
No that I can see. We were on trusty mainly because at some point xenial was bugged, but there should be no other reason than this.